### ISSUE TITLE
HyperOS 2 compatibility?

### ISSUE THREAD

**Issue by MikuDroid** [JonesqPacMan/android_device_xiaomi_aristotle_twrp#5]
Since hyperOS2 on 13T is still using same KMI as hyperOS1 (5.10.xxx-android12-9), does that means the latest released version still works with hyperOS2? Or the blobs will be something that prevent it to be compatible?

Or, the KMI does not even matter because you built the recovery based on HyperOS OS1.0.2.0.UMFMIXM blobs so it will only work with rom that you listed on release page? (_because their blobs are similar, maybe?_)

**Comment by JonesqPacMan** [comment]
Of course, installing is now easier with the vendor_boot repack and unpack technique via flashable Zip so it can fit in any version. stay tuned.
